static/migrate.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en" ng-app="sgMigrationApp">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <meta name="author" content="">
+
+    <title>SparkPost: SendGrid Template Translation</title>
+
+    <link href="vendor/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
+    <link href="vendor/font-awesome/css/font-awesome.min.css" rel="stylesheet">
+    <link href="main.css" rel="stylesheet">
+
+    <link rel="icon" href="favicon-16.png" sizes="16x16" />
+    <link rel="icon" href="favicon-32.png" sizes="32x32" />
+    <link rel="icon" href="favicon-48.png" sizes="48x48" />
+    <link rel="shortcut icon" href="favicon.ico" />
+
+    <script src="vendor/angular/angular.js"></script>
+    <script src="vendor/angular-sanitize/angular-sanitize.js"></script>
+    <script src="vendor/jquery/dist/jquery.min.js"></script>
+    <script src="vendor/bootstrap/dist/js/bootstrap.min.js"></script>
+    <script src="vendor/angular-bootstrap/ui-bootstrap-tpls.js"></script>
+    <script src="migrate.js"></script>
+  </head>
+  <body ng-controller="MigrateControl" ng-class="loading ? 'loading' : ''">
+    <div class="titlebar">
+      <a href="https://www.sparkpost.com">
+        <img class="navbar-logo" src="https://developers.sparkpost.com/images/logo-sparkpost-white.png" alt="SparkPost" width="150">
+      </a>
+    </div>
+    <div class="container-fluid">
+      <div class="jumbotron">
+        <div class="row">
+          <div class="col-md-8 col-md-offset-2">
+              <p>From here you can migrate a template from SendGrid to SparkPost.<br/>You will need a few things to hand to get started:</p>
+              <table class="table-migrate-notes">
+                <tr>
+                  <th class="pull-right">SendGrid API Key</th>
+                  <td>Requires <strong>Template Engine</strong> and <strong>Marketing Campaigns</strong> permissions</td>
+                </tr>
+                <tr>
+                  <th class="pull-right">SendGrid Template/Marketing Campaign ID</th>
```

Comment:
This is long and tricky to read. Let's say something "Template ID" on the left and just "Transactional template or campaign ID" on the right.